code/modules/atmospherics/machinery/binary/passive_gate.dm

```diff
@@ -59,9 +59,8 @@ obj/machinery/atmospherics/binary/passive_gate
 			if(network2)
 				network2.update = 1
 
-obj/machinery/atmospherics/binary/passive_gate/attackby(obj/weapon as obj, mob/user as mob)
-	..()
-	if(istype(weapon, /obj/item/device/multitool))
+obj/machinery/atmospherics/binary/passive_gate/attackby(obj/item/W as obj, mob/user as mob)
```

Comment:
`as obj` or `as mob` do nothing if this proc is not used as a verb. And one would hope it isn't.

Reply:
I'll fix that.